src/components/GetStarted/GetStarted.styles.ts

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import { pallete } from "../../misc/pallete";
+
+export const GetStartedWrapper = styled.div<{ mobileDisplay: boolean }>`
+  background-color: ${pallete.BrightRed};
+  color: ${pallete.VeryLightGray};
+  font-weight: 700;
+  padding: 12px 20px;
+  border-radius: 20px;
+  font-size: 12px;
+  &:hover {
+    background-color: ${pallete.VeryPaleRed};
+    cursor: pointer;
+  }
+  @media only screen and (max-width: 900px) {
```

Comment:
masz tutaj 2 zmienne do sterowania widokiem mobilnym - jedną z propsów (mobileDisplay) a jedną z media query (max-width: 900px). Jeśli nie chcesz wyświetlać tego komponentu w widoku mobilnym to wtedy ten props 'mobileDisplay' jest zbędny - samo media query będzie ok